backend/de.metas.business/src/main/sql/postgresql/system/20-de.metas.business/5697590_sys_gh15938_Mark_As_Fully_Delivered_2.sql

```diff
@@ -0,0 +1,235 @@
+;
+
+-- 2023-07-31T16:25:22.385Z
+INSERT INTO AD_Message_Trl (AD_Language,AD_Message_ID, MsgText,MsgTip, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy,IsActive) SELECT l.AD_Language, t.AD_Message_ID, t.MsgText,t.MsgTip, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy,'Y' FROM AD_Language l, AD_Message t WHERE l.IsActive='Y'AND (l.IsSystemLanguage='Y' OR l.IsBaseLanguage='Y') AND t.AD_Message_ID=545314 AND NOT EXISTS (SELECT 1 FROM AD_Message_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Message_ID=t.AD_Message_ID)
+;
+
+-- Value: de.metas.deliveryplanning.DeliveryPlanningService.OrderFullyDelivered
+-- 2023-07-31T16:26:01.968Z
+UPDATE AD_Message_Trl SET MsgText='Die Bestellung wird vollständig geliefert.',Updated=TO_TIMESTAMP('2023-07-31 19:26:01','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Language='de_CH' AND AD_Message_ID=545314
+;
+
+-- Value: de.metas.deliveryplanning.DeliveryPlanningService.OrderFullyDelivered
+-- 2023-07-31T16:26:04.708Z
+UPDATE AD_Message_Trl SET MsgText='Die Bestellung wird vollständig geliefert.',Updated=TO_TIMESTAMP('2023-07-31 19:26:04','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Language='nl_NL' AND AD_Message_ID=545314
+;
+
+-- Value: de.metas.deliveryplanning.DeliveryPlanningService.OrderFullyDelivered
+-- 2023-07-31T16:26:08.041Z
+UPDATE AD_Message_Trl SET MsgText='Die Bestellung wird vollständig geliefert.',Updated=TO_TIMESTAMP('2023-07-31 19:26:08','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Language='de_DE' AND AD_Message_ID=545314
+;
+
+-- Name: C_Order Not Fully Delivered
+-- 2023-07-31T16:45:09.475Z
+INSERT INTO AD_Val_Rule (AD_Client_ID,AD_Org_ID,AD_Val_Rule_ID,Created,CreatedBy,EntityType,IsActive,Name,Type,Updated,UpdatedBy) VALUES (0,0,540650,TO_TIMESTAMP('2023-07-31 19:45:09','YYYY-MM-DD HH24:MI:SS'),100,'D','Y','C_Order Not Fully Delivered','S',TO_TIMESTAMP('2023-07-31 19:45:09','YYYY-MM-DD HH24:MI:SS'),100)
+;
+
+-- Name: C_Order Not Fully Delivered
+-- 2023-07-31T16:48:53.764Z
+UPDATE AD_Val_Rule SET Code='C_Order.OrderStatus != ''FD''',Updated=TO_TIMESTAMP('2023-07-31 19:48:53','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Val_Rule_ID=540650
+;
+
+-- Name: C_Order Not Completely Invoiced
+-- 2023-07-31T16:50:21.349Z
+INSERT INTO AD_Val_Rule (AD_Client_ID,AD_Org_ID,AD_Val_Rule_ID,Code,Created,CreatedBy,EntityType,IsActive,Name,Type,Updated,UpdatedBy) VALUES (0,0,540651,'C_Order.InvoiceStatus != ''CI''',TO_TIMESTAMP('2023-07-31 19:50:21','YYYY-MM-DD HH24:MI:SS'),100,'D','Y','C_Order Not Completely Invoiced','S',TO_TIMESTAMP('2023-07-31 19:50:21','YYYY-MM-DD HH24:MI:SS'),100)
+;
```

Comment:
are those used? from what I've seen below there are not, am I missing somthing?

Reply:
@cp-ps yeah, these ones are removed below, I removed them manually to not mess up the migration

Reply:
@cp-ps I cleaned it up